source/_components/sensor.waqi.markdown

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: "World Air Quality Index"
+description: "Instructions how to setup World Air Quality Index sensor in Home Assistant."
+date: 2016-11-17 06:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: smtp.png
+ha_category: Sensor
+ha_release: 0.32
+ha_iot_class: "Local Polling"
+---
+
+The `waqi` sensor platform will query [World Air Quality Index](http://aqicn.org) service to check AQI value for a specific set of locations. The resulting indexes will be added to the Home Assistant as sensor outputs.
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: waqi
+    locations:
+      - beijin
+```
+
+Configuration variables:
+
+- **locations** (*Required*): a list of location names to look for air quality data. In case a specific location has multiple registered stations all of them will be added to Home Assistant
+
+The value reported is an overall AQ index for the location. The values of the index can be interpreted as following:
+
+- from 0 to 50 - **Good**. Air quality is considered satisfactory, and air pollution poses little or no risk,
```

Comment:
@Landrash I would rather have it as a table as on the original web site, however for some reason my markdown syntax wasn't good enough.